### ISSUE TITLE
right toolbar on Firefox / Ubuntu Linux

### ISSUE THREAD

**Issue by krisek** [jgraph/drawio#975]
### Preflight Checklist

* [x ] I agree to follow the [Code of Conduct](https://github.com/jgraph/drawio/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x ] I have searched the issue tracker for a feature request that matches the one I want to file, without success.

**Describe the bug**
The right hand side toolbar (Diagram|Style/Text/Arrange) doesn't render properly in Firefox on Ubuntu Linux.

**To Reproduce**
1. Create a completly new / clean Firefox profile so that no add-ons, existing settings interfere.
2. Open any diagram
3. Click on a shape

**Expected behavior**
The right hand side toolbar should render properly.

**Screenshots**
https://imagebin.ca/v/5PmEi7MYrfsI
https://imagebin.ca/v/5PmEsjWD8nB4
https://imagebin.ca/v/5PmEzuk25maI

**draw.io version (In the Help->About menu of the draw.io editor):**

- draw.io version 13.2.3

**Desktop (please complete the following information):**
 - OS: Ubuntu 20.04 LTS
 - Browser: Firefox
 - Version: 77.0.1

**Comment by alderg** [comment]
Unable to repeat in Firefox 77.0.1 and Ubuntu 20.04 LTS with 13.2.3. What are your font settings in about:preferences?

**Comment by krisek** [comment]
I was very surprised myself too, that's why I started with an empty profile so that nothing in my setup interfere. Now I created a screencast showing font preferences as well:

https://drive.google.com/file/d/10-zyRE7Fg4Mb_fglIqd2kViTVnYoP5wn/view?usp=sharing

Anyway... lleave it closed, I tested on an other system now and it is all good. I need to dig into this more and try to figure where is the problem on this particular system. I'll report here if I find something.

Many thanks for looking into it so far, much appreciated.
